pkg/remoteimage/pull.go

```diff
@@ -28,6 +29,16 @@ type puller struct {
 	keyring  credentialprovider.DockerKeyring
 }
 
+// Returns the size of the image that was pulled in MB(I think?) **TODO: check**
+func (p puller) ImageSize(ctx context.Context) (int, error) {
+	info, err := p.imageSvc.ImageFsInfo(ctx, &cri.ImageFsInfoRequest{})
```

Comment:
```suggestion
	info, err := p.ImageSize(ctx)
```

`ImageFSInfo` returns information of the filesystem that is used to store images (based on description), and `ImageSize` returns the size of the image that was pulled in MB (probably, even docs are not sure of units 😄).